tests/integration_test.py

```diff
@@ -358,7 +359,10 @@ def runTest(self):
         self.client.rename(res, name)
         inspect = self.client.inspect_container(res['Id'])
         self.assertIn('Name', inspect)
-        self.assertEqual(name, inspect['Name'])
+        if version == '1.5.0':
```

Comment:
would it make sense to use [compare_version](https://github.com/docker/docker-py/blob/master/docker/utils/utils.py#L99) here? so when 1.6 lands, one does not need to change the test again.